sdk/core/azure-core/CLIENT_LIBRARY_DEVELOPER.md

```diff
@@ -269,7 +269,7 @@ class HttpResponse(object):
     def text(self, encoding=None):
         """Return the whole body as a string."""
 
-    def stream_download(self, chunk_size=None, callback=None):
+    def stream_download(self, pipeline, **kwargs):
```

Comment:
QQ: what was the callback parameter? I'm pretty sure this was a doc bug, correct?

Reply:
Yes. It was a doc bug.